packages/services/f-feature-management/test/vue/features.service.test.js

```diff
@@ -0,0 +1,70 @@
+import FeaturesService from '../../src/vue/services/features.service';
+
+const mockStore = {
+    state: {
+        configuration: {
+            settings: {
+                featureManagement: {
+                    scope: 'je-coreweb',
+                    environment: 'development',
+                    obfuscationKey: 'abcdefghij0123456789',
+                    pollInterval: 1000 * 15
+                }
+            }
+        }
+    },
+    registerModule: jest.fn(),
+    dispatch: jest.fn()
+};
+
+const mockResponse = {
+    createdAt: '2021-09-10 15:00',
+    features: [
+        {
+            key: 'key1',
+            testVal: 'val1'
+        }
+    ]
+};
+
+const mockHttpClient = {
+    get: jest.fn(() => Promise.resolve({ data: mockResponse }))
+};
+
+const mockCookies = {};
+
+const mockAnalytics = {};
+
+const mockLogger = {
+    logInfo: jest.fn()
+};
+
+function createSUT () {
+    return new FeaturesService(mockStore, {
+        httpClient: mockHttpClient,
+        cookies: mockCookies,
+        analytics: mockAnalytics,
+        logger: mockLogger
+    });
+}
```

Comment:
If you're not referencing `this` anywhere you could convert this to an arrow function to keep it consistent with other function definitions?

```suggestion
const createSUT = () => new FeaturesService(mockStore, {
    httpClient: mockHttpClient,
    cookies: mockCookies,
    analytics: mockAnalytics,
    logger: mockLogger
});
```

Reply:
Although looking through this project it looks like a lot of the functions are defined using the `function` keyword... I thought we had a lint rule that pops a warning when doing that if you're not using `this` 🤔